### PR TITLE
feat: royalties

### PR DESCRIPTION
**Pull request by cazala**
Added support for `MarketplaceV2` with royalties 🏆

### CODE REVIEW

File: webapp/src/modules/nft/hooks.ts [hunk 1]
```diff
@@ -49,7 +48,7 @@ export const useComputedPrice = (nft: NFT, order: Order | null) => {
 
       setIsLoading(true)
       tokenConverter
-        .contractEthToMANA(order.ethPrice!)
+        .contractEthToMANA((order as Order & { ethPrice: string }).ethPrice!)
```
Comment (fzavalia) [inline webapp/src/modules/nft/hooks.ts:51]: Why is `ethPrice` not originally in the object.

**Review by fzavalia — COMMENTED** [review]
There is a bunch of repetition on extending the Order, can't that be solved with some sort of intermediate type to the commons one?

File: webapp/src/modules/nft/hooks.ts (hunk 1 above)
Comment (cazala) [reply webapp/src/modules/nft/hooks.ts:51]: This is something that only exists on partners, and the ideas is to remove it sooner than later, that's why I went the ugly and easy way to solve it :P

**Review by fzavalia — APPROVED** [review]

File: webapp/src/modules/nft/hooks.ts (hunk 1 above)
Comment (fzavalia) [reply webapp/src/modules/nft/hooks.ts:51]: Fair enough

**Pull request merged**